### PR TITLE
[16.0][ADD] delivery_package_type_number_parcels

### PR DESCRIPTION
**Pull request by glitchov**
Adds the number of parcels for a package type.

### CODE REVIEW

**Review by lmignon — APPROVED** [review]
Thank you @glitchov LGTM (Code review + functional tests)

Usefull addon to ease the management of the put in pack operation by allowing to create package type by number of parcels. In this way it becomes easy to scan a package type depending on the number of parcels into the package and automatically get the number of parcels information filled on the package.

**Review by jbaudoux — APPROVED** [review]

File: delivery_package_type_number_parcels/models/stock_package_type.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class PackageType(models.Model):
+    _inherit = "stock.package.type"
+
+    number_of_parcels = fields.Integer("Number of parcels")
```
Comment (rousseldenis) [inline delivery_package_type_number_parcels/models/stock_package_type.py:10]: @glitchov @jbaudoux Maybe something has to be more explained (in help here or in README) about what is really the number of parcels for a package type.

**Review by rousseldenis — APPROVED** [review]

File: delivery_package_type_number_parcels/tests/test_package_type_number_parcels.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.addons.stock.tests.test_packing import TestPackingCommon
+
+
+class TestStockQuantPackageDelivery(TestPackingCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```
Comment (AnizR) [inline delivery_package_type_number_parcels/tests/test_package_type_number_parcels.py:10]: can you maybe use  ? cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))

**Review by AnizR — APPROVED** [review]
Code LGTM besides my comment

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**